packages/components/app/styles/components/side-nav/toggle-button.scss

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// SIDE-NAV > TOGGLE BUTTON
+//
+
+
+.hds-side-nav__toggle-button {
+  position: absolute;
+  top: -2px;
+  left: calc(var(--token-side-nav-wrapper-border-width) * -1);
+  z-index: 1;
+  display: flex;
+  box-sizing: content-box; // keeps the border outside the element to consistently control the border width across SideNav elements
+  padding: calc(var(--token-side-nav-toggle-button-padding-vertical) - var(--token-side-nav-wrapper-border-width)) var(--token-side-nav-toggle-button-padding-horizontal);
+  color: var(--token-color-foreground-high-contrast);
+  background: none;
+  background-color: var(--token-side-nav-color-surface-primary);
+  border: var(--token-side-nav-wrapper-border-width) solid var(--token-side-nav-wrapper-border-color);
+  border-left-color: transparent;
+  border-top-right-radius: var(--token-side-nav-toggle-button-border-radius);
+  border-bottom-right-radius: var(--token-side-nav-toggle-button-border-radius);
+  transform:
+    translateX(var(--hds-app-sidenav-width-expanded))
+    translateY(24px);
+  cursor: pointer;
+  transition:
+    transform var(--hds-app-sidenav-animation-duration)
+    var(--hds-app-sidenav-animation-easing),
+    padding var(--hds-app-sidenav-animation-duration)
+    var(--hds-app-sidenav-animation-easing);
+
+  &::before {
+    position: absolute;
+    top: calc(var(--token-side-nav-toggle-button-border-radius) * -2);
+    left: calc(var(--token-side-nav-wrapper-border-width) * -1);
+    box-sizing: border-box;
```

Comment:
a couple of considerations:
- even if I comment this out, nothing changes (which makes sense, since with this implementation the sizing is given by content + padding + border, not by an explicit width/height)
- if you try to change the border width variable, the border is applied internally, not externally, which is not (I suspect) what the designers want

what I suggest to do (not a blocker, just for consideration): 
- get rid of the `box-sizing: border-box;`
- use explicit width/height for the element (see other comment)
I _think_ this _may_ simplify a bit the overall code (but I may be wrong, so take this with a pinch of salt)

Reply:
Removed the box-sizing declaration and the padding and moved to width/height declarations + flex alignment.